src/mono/sample/wasm/browser-bench/String.cs

```diff
@@ -16,9 +16,6 @@ class StringTask : BenchTask
         public StringTask()
         {
             measurements = new Measurement[] {
-                new NormalizeMeasurement(),
```

Comment:
Let's keep the benchmarks anyway

Reply:
Perhaps with smaller buffers, so that they are fast.

Reply:
like 8kB?